src/atoms/TextInput/Label.native.js

```diff
@@ -257,11 +267,15 @@ RegularLabel.propTypes = {
     width: PropTypes.number,
     x: PropTypes.number,
     y: PropTypes.number,
-  }).isRequired,
+  }),
+  labelPosition: PropTypes.oneOf(['left', 'top']).isRequired,
+  disabled: PropTypes.bool,
 };
 
 RegularLabel.defaultProps = {
   children: 'Label',
+  inputLayoutDimensions: undefined,
```

Comment:
Won't this be `undefined` by default since `props` is an object?

Reply:
You can also keep this `{}`.

Reply:
We can, but I don't see the benefit of making it an object when it is actually undefined on initial render

Reply:
Even if we don't explicitly mention in `defaultProps`, wouldn't it be `undefined` anyway?

Reply:
We decided to explicitly make it undefined for all non-```isRequired``` props. We're doing this everywhere

Reply:
Cool, makes sense. 👍